### PR TITLE
added Util object for inferring schema

### PR DESCRIPTION
**Pull request by praveenkumarb1207**

### CODE REVIEW

File: src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.github.music.of.the.ainur.almaren
+
+import org.apache.spark.sql.DataFrame
+
+object Util {
+  val spark = Almaren.spark.getOrCreate()
+
+  import spark.implicits._
+
+  def genDDLFromJsonString(df: DataFrame, field: String): String = {
```
Comment (mantovani) [inline src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala:10]: The "sample" should be coming function's argument.

File: src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  val spark = Almaren.spark.getOrCreate()
+
+  import spark.implicits._
+
+  def genDDLFromJsonString(df: DataFrame, field: String): String = {
+
+    spark.read.json(df.select(field).as[String]).sample(0.1).schema.toDDL
+  }
+
+  def genDDLFromDataFrame(df: DataFrame): String = {
```
Comment (mantovani) [inline src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala:15]: same here

**Review by mantovani — CHANGES_REQUESTED** [review]
We also need to add a new section to the documentation.

File: src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala (hunk 2 above)
Comment (praveenkumarb1207) [reply src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala]: completed the changes Daniel

File: src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.github.music.of.the.ainur.almaren
+
+import org.apache.spark.sql.DataFrame
+
+object Util {
+  val spark = Almaren.spark.getOrCreate()
+
+  import spark.implicits._
+
+  def genDDLFromJsonString(df: DataFrame, field: String,sampleRatio : Double): String = {
```
Comment (mantovani) [inline src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala:10]: Let's have sampleRatio = 1.0 as default.

**Review by mantovani — CHANGES_REQUESTED** [review]

File: src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala (hunk 3 above)
Comment (praveenkumarb1207) [reply src/main/scala/com/github/music/of/the/ainur/almaren/Util.scala]: Updated sampleRatio = 1.0 as default.

**Review by mantovani — APPROVED** [review]

**Pull request merged**